monocle/github/pullrequest.py

```diff
@@ -386,24 +405,44 @@ def extract_pr_objects(pr):
                     'created_at': _commit.get('pushedDate', change['created_at']),
                 }
                 if _commit['author'].get('user'):
-                    obj['author'] = _commit['author']['user'].get('login')
+                    obj['author'] = get_login(_commit['author']['user'])
                 insert_change_attributes(obj, change)
                 objects.append(obj)
             return objects
 
+        def dump_data(data, prefix=None):
```

Comment:
Do you think that's possible to have the same dump support for the Gerrit crawler ?

Reply:
sure but let's do that in another PR

Reply:
Ok. So let's merge this PR. I'm adding an issue to not forget about adding the same support for the Gerrit crawler.

Reply:
done